configuration_sample.yaml

```diff
@@ -0,0 +1,16 @@
+# Sample configuration to use with `scripts/apply_model.py`.
+
+run_model_func: pv_site_production.models.psp.psp_model.get_model
+model_path: path/to/model.pkl
+
+nwp:
+  cls: pv_site_production.data.nwp_data_sources.NwpDataSource
+  kwargs:
+    path: s3://nowcasting-nwp-development/data/latest.netcdf
+    storage_kwargs:
+      key: <aws key>
+      secret: <asw secret>
+
+pv_metadata_path: ./meta_inferred.csv
+
+pv_db_url: 'postgresql+psycopg2://<user>:<password>@<host>:<port>/<db_name>'
```

Comment:
Ive done things before by setting ENV VARs in the configuration. This might be worth doing, as the nwp and pv data will be different for development and production envs

E.G. https://github.com/openclimatefix/nowcasting_forecast/blob/main/nowcasting_forecast/config/mvp_v2.yaml#L21

Reply:
Sure it can be environment variables. This is because you want to use the *same* config in both environments?

In this case:
```
 nwp_zarr_path: !ENV ${NWP_ZARR_PATH}
```

Where is `!ENV ${NWP_ZARR_PATH}` replaced with the actual value?

Reply:
so in the tests, you can set `NWP_ZARR_PATH` as a fixture, for example

in live, this can be set as an ENVVAR, and is set in `terraform` code.

Does that make sense?

Reply:
I understand the principle but I'm confused with the specifics in our case!
I did some work to make the `testcontainers` work and had to do something similar where I have placeholders in the config. It is similar to this but probably a slightly different implementation. Let's keep the discussion for that next PR if it's ok with you.